modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/graphql/dto/v1_0/ObjectDefinitionGraphQLDTOContributor.java

```diff
@@ -182,7 +183,10 @@ dtoConverterContext, _objectDefinition, _toObjectEntry(dto),
 
 	@Override
 	public boolean deleteDTO(long id) throws Exception {
-		_objectEntryManager.deleteObjectEntry(_objectDefinition, id);
+		DefaultObjectEntryManager defaultObjectEntryManager =
```

Comment:
Can we move this call to DefaultObjectEntryManagerProvider.provide to the constructor of this class? this way we avoid to call it many times in many methods.
Maybe our private variable should be the type DefaultObjectEntryManager (if we didn't call methods from ObjectEntryManager on this class).

Reply:
@guilhermedcamacho,

in this class we call [this method](https://github.com/liferay-objects/liferay-portal/blob/56d8c2cbeefa7b94469b68131a42b849db3f09ca/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/graphql/dto/v1_0/ObjectDefinitionGraphQLDTOContributor.java#L218) that doesn't belong to DefaultObjectEntryManager due to this I'm calling DefaultObjectEntryManagerProvider.provide for each method.